static/skin/i18n/en.json

```diff
@@ -44,12 +44,12 @@
 	, "search-results-page-title": "Search: {{SEARCH_PATTERN}}"
 	, "search-results-page-header": "Results <b>{{START}}-{{END}}</b> of <b>{{COUNT}}</b> for <b>\"{{{SEARCH_PATTERN}}}\"</b>"
 	, "empty-search-results-page-header": "No results were found for <b>\"{{{SEARCH_PATTERN}}}\"</b>"
-	, "search-result-book-info": "from {{BOOK_TITLE}}"
+	, "search-result-book-info": "from {{{BOOK_TITLE}}}"
```

Comment:
Unlike the other two translations that are used to set DOM attributes via JS, this one is used when rendering an HTML or XML template so it should undergo HTML encoding, otherwise book titles containing text that looks like a HTML tag (esp. `</div>`) will result in invalid HTML on the search results page. You can check that this change isn't needed by selecting the bottommost "containing 'searchterm'..." suggestion and paying attention to the "from BookTitle" line below each search result.

Reply:
@veloman-yunkan I have made an other fix, because like I suspected we face here a double encoding by Mustache.js.... Not sure what is the best, but pretty sure we have other problem of that kind in other places. What do you think?